src/CMakeLists.txt

```diff
@@ -364,8 +364,8 @@ endif()
 #=================== Linking ===================
 
 target_link_libraries(libultraship
-    PRIVATE ZAPDUtils StrHash64
-    PUBLIC ImGui storm tinyxml2 nlohmann_json::nlohmann_json
+    PRIVATE StrHash64
+    PUBLIC ZAPDUtils ImGui storm tinyxml2 nlohmann_json::nlohmann_json
```

Comment:
needed this because things couldn't find `Vec2f`, not sure why that was a wii u only issue...